corehq/apps/settings/languages.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+
+from corehq.apps.users.views import get_domain_languages
+
+
+def get_languages_for_user(user):
+    """
+    Returns all CommCare supported languages and any languages that have
+    explicitly been added to domains this user is a member of as a list
+    of tuples list((language code, display name))
+    """
+    translated_languages = get_translated_languages()
+    domain_languages = []
+    for domain_membership in user.domain_memberships:
+        domain_languages.extend(get_domain_languages(domain_membership.domain))
+
+    return sorted(set(translated_languages + domain_languages))
```

Comment:
Is it possible for `translated_languages` and `domain_languages` to have different display names for a given language, and therefore this list could have duplicates? Is it worth deduping on the language code?